.changeset/famous-radios-watch.md

```diff
@@ -0,0 +1,5 @@
+---
+"comet-authproxy-v2": major
```

Comment:
why is this a major change? shouldnt this be a minor?

Reply:
Version was changed.

70a3bf7b68720edb0262060b13fdf7b0ea8adad3

Reply:
Changed charset doc 

59bc0fc1444e1eb8e24576735984db0a1323c71f